userguide/tutorials/aws-standalone.adoc

```diff
@@ -69,36 +69,40 @@ Also when starting the AMIs, you will need to specify the new properties for you
 ```
 #!/bin/bash
```

Comment:
Regarding simplification I toyed with a few different ideas, but in the end, it's always complicated...

So I would suggest to keep this as-is, but simplify the example to avoid extra functions, variables, etc.

```
#!/bin/bash

cat <<_EOF > /var/tmp/db.props
#
# EDIT FOLLOWING DB PROPERTIES BELOW AS NEEDED
#
DB_SERVER="dbserver:3306"
DB_USER="myuser"
DB_PASSWORD="mypwd"
KILLBILL_DB_NAME="killbill"
KAUI_DB_NAME="kaui"
_EOF

cat <<_EOF > /var/tmp/kb.props
#
# EDIT FOLLOWING KB PROPERTIES BELOW AS NEEDED
#
org.killbill.dontexist=foo
_EOF

su -l -c "cd /var/lib/tomcat/bin && /var/lib/tomcat/bin/updateProperties.sh /var/tmp/db.props /var/tmp/kb.props" tomcat
```